projects/storefrontstyles/scss/themes/calydon/layout/header/_header.scss

```diff
@@ -0,0 +1,10 @@
+%header-calydon {
+  .SiteLogo {
+    a:focus {
+      display: block;
```

Comment:
we shouldn't check something that crucial as display property on events. can you explain what's going on here?

Reply:
Firefox patch, otherwise it won't show.

Reply:
setting `display` directly to anchor, seems we don't have any cross-browser affectation, so this could be a better approach.